### DISCUSSION TITLE
Deployment video tutorial 😍

### DISCUSSION THREAD

**Discussion by rinrinx2** [spencerwooo/onedrive-vercel-index#262] [Show and tell]
https://youtu.be/47cVvI2Kw1M

**Comment by spencerwooo** [comment]
thanks for the effort, i have converted this as an extra tutorial in the discussion forum and labelled it for you. 😊

**Reply by rinrinx2** [reply]
New video edu login bro coming

**Reply by hirockstar** [reply]
do it fast bro... please

**Reply by rinrinx2** [reply]
i will upload it today

**Reply by hirockstar** [reply]
@kamileecher Great Brother Im waiting...
